### DISCUSSION TITLE
paglib是否支持x86架构

### DISCUSSION THREAD

**Discussion by 147API** [Tencent/libpag#1731] [Q&A] [answered]
paglib是否支持x86架构

**Comment by kevingpqi123** [comment]
pag 的所有源码都是开源的，需要什么架构自己编译即可

**Reply by domchen** [reply, marked as answer]
如果是安卓端，我们的release版本没有预先构建x86架构的，只有arm和arm64架构。因为x86架构的也可以自动使用arm架构的包。源码应该也编译不出来，主要是Android端依赖的第三方库也没有写编译x86架构的脚本。目前基本没有遇到过这类需求，需要自己改造扩展一下。如果不是Android端，其他平台自己编译就行。
